apps/base-docs/docs/tools/network-faucets.md

```diff
@@ -9,6 +9,8 @@ keywords:
     testnet ETH,
     Base testnet,
     Coinbase Faucet,
+    Coinbase Developer Platform,
+    cdp faucet,
```

Comment:
nit: CDP Faucet?

Reply:
I think keeping as lowercase makes more sense - I don't think people would capitalize CDP in their search

we have `testnet resources` (all lowercase) as a keyword

Reply:
hmm I saw all these other faucet are in Caps though. Will probably wait for a 3rd opinion here.

@zencephalon thoughts?

Reply:
in the new diff I've added `Coinbase Developer Platform Faucet` but this would be the abbreviated version `cdp faucet` which should be a more common search

Reply:
not sure how case sensitive these keywords are but guessing some of the other ones are uncapitalized for a reason

Reply:
I don't expect these keywords to exhibit case sensitivity.